utils/clients/tests/test_django_database_client.py

```diff
@@ -95,3 +95,26 @@ def test_get_instrument_invalid(self):
         client.connect()
         actual = client.get_instrument('Fake instrument')
         self.assertIsNone(actual)
+
+    # pylint:disable=no-self-use
+    def test_get_reduction_run_valid(self):
+        """
+        Test: A ReductionRun record is returned
+        When: get_reduction_run is called with values that match a database record
+        """
+        mock_data_model = Mock()
+        client = DatabaseClient()
+        client.connect()
+        client.data_model = mock_data_model
+        client.get_reduction_run('GEM', 1)
```

Comment:
Should we test that the return value of this is something?

Reply:
It's a mocked value, so it does not actually return anything. We don't want to use the real database as it will fail locally without the test database